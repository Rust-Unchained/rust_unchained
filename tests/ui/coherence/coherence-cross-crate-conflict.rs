//@ aux-build:trait_impl_conflict.rs

extern crate trait_impl_conflict;
use trait_impl_conflict::Foo;

// Conflicting implementations, upstream defines Foo for isize
impl<A> Foo for A { //~ ERROR E0119
}

fn main() {
}
