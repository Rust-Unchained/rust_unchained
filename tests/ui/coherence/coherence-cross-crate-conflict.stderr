error[E0119]: conflicting implementations of trait `Foo` for type `isize`
  --> $DIR/coherence-cross-crate-conflict.rs:7:1
   |
LL | impl<A> Foo for A {
   | ^^^^^^^^^^^^^^^^^
   |
   = note: conflicting implementation in crate `trait_impl_conflict`:
           - impl Foo for isize;

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0119`.
